src/Integration/MefServices/ActiveSolutionTracker.cs

```diff
@@ -38,7 +38,7 @@ internal sealed class ActiveSolutionTracker : IActiveSolutionTracker, IVsSolutio
         /// <summary>
         /// <see cref="IActiveSolutionTracker.ActiveSolutionChanged"/>
         /// </summary>
-        public event EventHandler ActiveSolutionChanged;
+        public event EventHandler<bool> ActiveSolutionChanged;
```

Comment:
We have a rule saying with should always use a type inheriting from `EventArgs` as payload of an event so it might be better to comply with it here.